### PR TITLE
Add navigation bar, experience, and customize front page

### PR DESCRIPTION
**Pull request by dhruvi618**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,31 +1,65 @@
...
+                <img src="images/rutgers.png" alt="iCIMS" width="100" height="100">
+                <h2>iCIMS</h2>
+                <p>SWE Intern</p>
+                <p>Java Backend</p>      
+            </div>
+        </section>
+        <section id="contact">   
+            <div class="icons"> 
+                <a href="https://www.linkedin.com/in/dhruvi-sheth">
+                    <img alt="Linkedin" src="images/linkedin.png" width="30" height="25">
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html:48]: 😄  Nice job including an alt-text here. Can you expand it a little bit to explain more precisely where this link goes. Maybe "Dhruvi Sheth's LinkedIn Profile"

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,31 +1,65 @@
...
+                </a>
+            </div>
+            <div class="icons">
+                <a href="https://github.com/dhruvi618">
+                    <img alt="Github" src="images/github.png" width="25" height="25">
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html:53]: same comment about alt text

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,31 +1,65 @@
...
+                </a>
+            </div>
+            <p>Click here to get a random message:</p>
+            <button onclick="addRandomGreeting()">Hello</button>
+            <div id="greeting-container"></div>
+        </section>
+        </br>
+        <!--<p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
+        <div id="greeting-container"></div>-->
```
Comment (kmh287) [inline portfolio/src/main/webapp/index.html:63]: delete commented out code?

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,31 +1,65 @@
...
+            <a class="active" href="#home">Home</a>
+            <a href="#">About Me</a>
+            <a href="#">Experience</a>
+            <a href="#">Projects</a>
+            <a href="#">Contact</a>
+        </div>
+        <section id="home">
+            <div>
+                <h1>I'm Dhruvi Sheth.</h1>
+                </br>
```
[the remark this answers is not in the document]
Comment (dhruvi618) [reply portfolio/src/main/webapp/index.html:20]: Yes, this break was just for additional spacing

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (kmh287) [reply portfolio/src/main/webapp/index.html:48]: Need to git push?

File: portfolio/src/main/webapp/index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (kmh287) [reply portfolio/src/main/webapp/index.html:20]: Is there any way the additional spacing can be added as CSS for `<h1>`? IIRC `<br>` is generally discouraged if only used for spacing.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (dhruvi618) [reply portfolio/src/main/webapp/index.html]: just pushed the changes

File: portfolio/src/main/webapp/index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (dhruvi618) [reply portfolio/src/main/webapp/index.html]: removed the <br> and added padding through CSS instead

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,14 +1,86 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+* {
+  margin:0;
```
Comment (tcwanggoogle) [inline portfolio/src/main/webapp/style.css:2]: It seems like indent=4 for other parts of this file?

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -1,14 +1,86 @@
...
+}
+
+.icons {
     margin: 5px;
     float: left;
+    bottom: 50px;
 }
+
+#greeting-container {
+  margin-top: 20px;
```
Comment (tcwanggoogle) [inline portfolio/src/main/webapp/style.css:85]: Same comment about the indent

**Pull request merged**